plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveGlueMetadataListing.java

```diff
@@ -98,7 +100,12 @@ public void cleanup()
     @Test
     public void testReadInformationSchema()
     {
-        String expectedTables = format("VALUES '%s', '%s', '%s'", TpchTable.REGION.getTableName(), TpchTable.NATION.getTableName(), FAILING_TABLE_WITH_NULL_STORAGE_DESCRIPTOR_NAME);
+        String expectedTables = format(
```

Comment:
We could use `computeActual("...").getOnlyColumnAsSet()` method in the following assertions. Thus, we can avoid  `skippingTypesCheck` there and redundant string concat here. No requested change as pre-existing.